go/vt/vtgate/planbuilder/builder.go

```diff
@@ -353,6 +357,35 @@ func createInstructionFor(query string, stmt sqlparser.Statement, vschema Contex
 	return nil, vterrors.Errorf(vtrpcpb.Code_INTERNAL, "BUG: unexpected statement type: %T", stmt)
 }
 
+func buildDBDDLPlan(stmt sqlparser.Statement, vschema ContextVSchema) (engine.Primitive, error) {
```

Comment:
`stmt` could be typed to `*sqlparser.DBDDL` in the function signature here

Reply:
this implements the function signature `f func(statement sqlparser.Statement, schema ContextVSchema)`